doc/yaml/db/produce_db.yml

```diff
@@ -0,0 +1,24 @@
+###########################################################################
+# Item Produce Database
+###########################################################################
+#
+# Item Produce Settings
+#
+###########################################################################
+# - ItemLevel                  Number which determines what kind of a crafting window will pop-up.
+#   Recipe:
+#    - Product                 AegisName of the produced item.
+#      SkillName               Skill name required. (Default: null)
+#      SkillLevel              Skill level required. (Default: 1)
+#      Consumed:               List of items consumed to produce the Product.
+#        - Item                AegisName of the consumed item.
+#          Amount              Amount required.
+#          Clear               Remove the item with the given AegisName from Consumed. (Optional)
```

Comment:
The item name is already defined in "item".
Would it not be better to make "Clear" a boolean true/false?